### PR TITLE
Add Sidekiq 6.1 as a supported Sidekiq version

### PR DESCRIPTION
**Pull request by agrobbin**
## What did we change?

Loosens the `sidekiq` gem dependency to allow `v6.1.x`.

## Why are we doing this?

https://github.com/mperham/sidekiq/blob/19cf129fe34a979ef2eb129a1e07fd83fb8a895a/Changes.md#610

## How was it tested?
- [x] Specs
- [ ] Locally
- [ ] Staging

### CODE REVIEW

File: sidekiq_publisher.gemspec [hunk 1]
```diff
@@ -58,5 +58,5 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency "activerecord-postgres_pub_sub", ">= 0.4.0"
   spec.add_runtime_dependency "activesupport", ">= 5.1", "< 6.1"
-  spec.add_runtime_dependency "sidekiq", ">= 5.0.4", "< 6.1"
+  spec.add_runtime_dependency "sidekiq", ">= 5.0.4", "< 6.2"
```
Comment (tjwp) [inline sidekiq_publisher.gemspec:61]: For Sidekiq I'd be okay with extending this to "< 7.0" but don't feel strongly.

Comment (agrobbin) [reply sidekiq_publisher.gemspec]: Works for me! Just updated this to be `< 7.0`.

**Comment by agrobbin** [comment]
@tjwp just updated this to add the new Circle CI steps, totally forgot about those!

**Comment by tjwp** [comment]
Thanks, I'll merge and cut a new release now!

**Comment by agrobbin** [comment]
Thanks @tjwp !!

**Comment by tjwp** [comment]
v1.6.4 released

**Pull request merged**